eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -254,8 +256,15 @@ function DeployStressPackage(
             Write-Host "Building and pushing stress test docker image '$imageTag'"
             $dockerFile = Get-ChildItem $dockerFilePath
 
-            Run docker build -t $imageTag -f $dockerFile $dockerBuildFolder
+            $dockerBuildCmd = @("docker"; "build"; "-t"; $imageTag;"-f";$dockerFile)
+            foreach ($envVar in $dockerBuildConfig.scenario.GetEnumerator()) {
```

Comment:
I'm still paranoid about the weird null checking issues we've been seeing, so am wondering if we'll see them again.

Reply:
This won't fail because in order for this step to execute, you'll already need dockerBuildConfig, which means scenarios is not empty.

Reply:
line 244 and 215 shows this ^